.gitignore

```diff
@@ -12,6 +12,9 @@ data/DECam_default_calibrators
 .pytest.ini
 tests/plots
 tests/temp_data
+webap/seechange_webap_config.py
+webap/static/rkwebutil.js
+webap/install
```

Comment:
Just curious: don't we need these files on the repo?

Reply:
I will change this.  webap/static/rkwebutil.js is made in the Makefile (it's just copied out of a submodule), so it should stay in .gitignore.  The other two are vestigal from before I edited the makefile and how these things were configured, so should generally not exist anyway.